### DISCUSSION TITLE
Consider a new field:  "Mention of (Print)" - Print

### DISCUSSION THREAD

**Discussion by AmirSimantov** [PanditProject/Website#17] [Help] [answered]
Hi, @mark-rozen. I am separating this question from issue #16. 

Your words:
> If it is obligatory, maybe we could add a specific field in the print entity that will connect between volumes of actually the same book? I think it’s good to have a specific field that will connect between print entities, in general. For example, I’ve tried to connect between these two Mohanty 1984 ([entity/63440/print](https://www.panditproject.org/entity/63440/print)) and Potter 1984 ([entity/63441/print](https://www.panditproject.org/entity/63441/print)), but I guess I did wrong because I used the field "Discrete facts” which will no longer be supported soon.

If I understand correctly, one article mentions another article. If so, it may be in various ways, like citing, paraphrasing, alluding, etc. The mentioning Print entity may also judge the other Print entity (endorsing, critiquing).

We currently don't have such relationships among Print entities (we _do_ have similar fields in the Work entity type (see the configuration spreadsheet [Work tab](https://docs.google.com/spreadsheets/d/1SLjJyuPEU8BXfschex56HGRAA9M3AKjMWWlyq7cJf88/edit#gid=68)).

The relationships make sense. However, we should think carefully if we want to spread Pandit in this direction.

A discussion is welcome.

**Comment by ybronner** [comment, marked as answer]
@mark-rozen  it is good that you raised this idea, and please continue to bring up suggestions. However, in this case, we will veer into the endless ocean of bibliographical data. Each article or book sites numerous other print sources, and while this is of interest to a bibliographic data base, it is not in our domain, which is prosopographical.
